common/djangoapps/enrollment/data.py

```diff
@@ -293,3 +296,23 @@ def get_course_enrollment_info(course_id, include_expired=False):
         raise CourseNotFoundError(msg)
     else:
         return CourseSerializer(course, include_expired=include_expired).data
+
+def get_roster(course_id):
+    roster = []
+
+    course_key = CourseKey.from_string(course_id)
+    qset = CourseEnrollment.objects.filter(
+        course_id=course_key,
+        is_active=True
+    ).order_by('user__username')
+
+    roster = [{
+        'anonymous_user_id': anonymous_id_for_user(enrollment.user, None),
+        'user_id': enrollment.user.id,
+        'username': enrollment.user.username,
+        'email': enrollment.user.email,
+        # add name here somehow!!!
+        'mode': enrollment.mode
+    } for enrollment in qset if not user_has_role(enrollment.user, CourseStaffRole(course_key))]
```

Comment:
I can guess reasons why you may have done this, but I'm not sure we want this. CourseStaff are not universally excluded from queries, which means if you, for example, display a "head count" of enrolled users, your count will never be the same as one displayed in the LMS.

That's the kind of thing that gets us support tickets, "hey why are these counts different?!", so I'd generally like to avoid it, for that reason alone.

Is there a compelling reason to exclude them from the list?